parser/src/main/kotlin/com/linkedin/dex/parser/DexParser.kt

```diff
@@ -21,17 +26,23 @@ import java.util.zip.ZipInputStream
  * NOTE: everything in the spec package is derived from the dex file format spec:
  * https://source.android.com/devices/tech/dalvik/dex-format.html
  */
-class DexParserArgs(parser: ArgParser) {
-    val apkPath by parser.positional(
-        help = "path to apk file"
-    )
+private class DexParserCommand : CliktCommand() {
+    val apkPath: String by argument().help("path to apk file")
 
-    val outputDir by parser.positional(
-        help = "path to output dir where AllTests.txt file will be saved, if not set output will go to stdout"
+    val outputDir by argument().help("path to output dir where AllTests.txt file will be saved, if not set output will go to stdout"
     ).default("")
 
-    val customAnnotations by parser.adding(
-        "-A", help = "add custom annotation used by tests") { toString() }
+    val customAnnotations: List<String> by option("-A", "--annotation").multiple().help("add custom annotation used by tests")
+
+    override fun run() {
+        println("Got file $apkPath output dir $outputDir, annotations: q${customAnnotations}q")
```

Comment:
Will the additional console output break any consuming scripts?

Reply:
Oh you're right, i meant to remove this woops